frontend/src/lib/api/generated/types.gen.ts

```diff
@@ -519,7 +519,7 @@ export type PostPaymentData = {
          */
         metadata?: string;
         /**
-         * The a unique nounce from the purchaser. Required to be in hex format
+         * The a unique nonce from the purchaser. Required to be in hex format
```

Comment:
The sentence doesn't make sense. 

```suggestion
         * A unique nonce from the purchaser, needs to be in hex format.
```